web/conf.py

```diff
@@ -23,10 +23,10 @@
 # -- Links --
 # This is the main URL for your site. It will be used
 # in a prominent link. Don't forget the protocol (http/https)!
-SITE_URL = "https://plasmapy.github.io/"
+SITE_URL = "https://plasmapy.org/"
```

Comment:
I had to change this up so the URLs in search would come out right.

... I hope it didn't break anything.